pkg/controller/seed-controller-manager/mla/project_controller.go

```diff
@@ -0,0 +1,231 @@
+/*
+Copyright 2021 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mla
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"strconv"
+
+	"go.uber.org/zap"
+
+	grafanasdk "github.com/kubermatic/grafanasdk"
+	"k8c.io/kubermatic/v2/pkg/controller/util/predicate"
+	kubermaticv1 "k8c.io/kubermatic/v2/pkg/crd/kubermatic/v1"
+	"k8c.io/kubermatic/v2/pkg/kubernetes"
+	"k8c.io/kubermatic/v2/pkg/version/kubermatic"
+
+	"k8s.io/client-go/tools/record"
+	"k8s.io/utils/pointer"
+	ctrlruntimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+)
+
+const (
+	grafanaOrgAnnotationKey = "mla.k8c.io/organization"
+)
+
+// projectReconciler stores necessary components that are required to manage MLA(Monitoring, Logging, and Alerting) setup.
+type projectReconciler struct {
+	ctrlruntimeclient.Client
+	grafanaClient *grafanasdk.Client
+
+	log        *zap.SugaredLogger
+	workerName string
+	recorder   record.EventRecorder
+	versions   kubermatic.Versions
+}
+
+// Add creates a new MLA controller that is responsible for
+// managing Monitoring, Logging and Alerting for user clusters.
+func newProjectReconciler(
+	mgr manager.Manager,
+	log *zap.SugaredLogger,
+	numWorkers int,
+	workerName string,
+	versions kubermatic.Versions,
+	grafanaClient *grafanasdk.Client,
+) error {
+	log = log.Named(ControllerName)
+	client := mgr.GetClient()
+
+	reconciler := &projectReconciler{
+		Client:        client,
+		grafanaClient: grafanaClient,
+
+		log:        log,
+		workerName: workerName,
+		recorder:   mgr.GetEventRecorderFor(ControllerName),
+		versions:   versions,
+	}
+
+	ctrlOptions := controller.Options{
+		Reconciler:              reconciler,
+		MaxConcurrentReconciles: numWorkers,
+	}
+	c, err := controller.New(ControllerName, mgr, ctrlOptions)
+	if err != nil {
+		return err
+	}
+
+	debugPredicate := predicate.ByLabel(kubermaticv1.WorkerNameLabelKey, workerName)
```

Comment:
This only works when you were implementing this controller and this would only reconcile Project with your worker label.
This should be removed to work with all projects in the system

Reply:
remove, thx